hw/xbox/nv2a/nv2a.c

```diff
@@ -312,12 +312,11 @@ static void nv2a_realize(PCIDevice *dev, Error **errp)
 
     dev->config[PCI_INTERRUPT_PIN] = 0x01;
 
-    d->pcrtc.start = 0;
-
-    d->pramdac.core_clock_coeff = 0x00011c01; /* 189MHz...? */
-    d->pramdac.core_clock_freq = 189000000;
+    d->pcrtc.start                = 0;
+    d->pramdac.core_clock_coeff   = 0x00011c01; /* 189MHz...? */
+    d->pramdac.core_clock_freq    = 189000000;
```

Comment:
I dislike the alignment here, because it hides that we mix hex and dec here.

Reply:
Huh? How does it hide that?